src/interpreter/Interpreter.cpp

```diff
@@ -732,7 +748,15 @@ ByteCodeStackOffset* Interpreter::interpret(ExecutionState& state,
         auto dstStart = readValue<int32_t>(bp, code->srcOffsets()[0]);
         auto value = readValue<int32_t>(bp, code->srcOffsets()[1]);
         auto size = readValue<int32_t>(bp, code->srcOffsets()[2]);
-        m->fill(state, dstStart, value, size);
+
+        if (!m->fill(dstStart, value, size)) {
+            std::string str = "out of bounds memory access: access at ";
+            str += std::to_string(dstStart + value);
+            str += "+";
+            str += std::to_string(size);
+            Trap::throwException(state, str);
```

Comment:
If you want to replace every exception as return true/false,
every code and function should return `the execution was good or there was a excpetion`

Reply:
We should check `there was a excpetion` on every site

Reply:
Would you prefer an `enum` instead of true/false?

Reply:
Yes. IMO returning enum is easy to understand

Reply:
I agree. Enums are just turned to 0/1 in the compiler, so they don't slow down the code.